source/Halibut/Transport/Streams/WriteIntoMemoryBufferStream.cs

```diff
@@ -0,0 +1,123 @@
+﻿using System;
+using System.IO;
+using System.Threading;
+using System.Threading.Tasks;
+using Halibut.Transport.Observability;
+
+namespace Halibut.Transport.Streams
+{
+    public class WriteIntoMemoryBufferStream : Stream
+    {
+        readonly MemoryStream memoryStream;
```

Comment:
nit: could be called buffer since it is a buffer...

Reply:
Sure.